docs/src/pages/[platform]/getting-started/shared/paintings.ts

```diff
@@ -0,0 +1,122 @@
+export const PAINTINGS = [
```

Comment:
should the file be name as `paintingData.ts` or put it in a data folder `data/paintings.ts`? I know it's for an example code, so we may want to have a higher bar even on naming